samples/snippets/pom.xml

```diff
@@ -187,6 +189,9 @@
             <spanner.sample.database>mysample</spanner.sample.database>
             <spanner.quickstart.database>quick-db</spanner.quickstart.database>
           </systemPropertyVariables>
+          <excludes>
+            <exclude>**/SpannerSampleIT.java</exclude>
+          </excludes>
```

Comment:
I understand that this one is slow, but do we really want to skip it all-together? And/or will we have some other nightly job that overrides this configuration and does include this test?

Reply:
I am introducing a new job to run this test alone in parallel so that it can be completed in an hour.